### DISCUSSION TITLE
API question

### DISCUSSION THREAD

**Discussion by af-gokhan** [magma/magma#10263] [Q&A] [answered]
Quick question about API: it is not clear if API is multi-tenant and how do you assign a certain certificate to an organization? I do not see any docs or examples of calling APIs externally either. Thanks!

**Comment by ulaskozat** [comment]
@hcgatewood, @arunuke do we have pointers for @af-gokhan on this?

**Comment by hcgatewood** [comment, marked as answer]
We don't currently support access permissions at the Orc8r API (*). However, usage through the NMS will partition networks by organization, which provides multi-tenancy

(*) Magma v1.7 is currently projected to support *per-network* access permissions at the Orc8r API. This isn't the same as how the NMS handles multi-tenancy, but the plan is to unify things by the v1.8 release

**Reply by af-gokhan** [reply]
Thanks for the info Hunter.

**Comment by edaspb** [comment]
Maybe I'm wrong and it is not working on current version but few releases ago it was possible manually create certificate with per-organization access restriction:
```
envdir /var/opt/magma/envdir /var/opt/magma/bin/accessc add -duration 3650 -n organization:RW -cert /var/opt/magma/bin/organization_operator organization_operator
openssl pkcs12 -export -out /var/opt/magma/bin/organization_operator.pfx -inkey /var/opt/magma/bin/organization_operator.key.pem -in /var/opt/magma/bin/organization_operator.pem
```
